test/box-luatest/gh_8663_persistent_triggers_test.lua

```diff
@@ -0,0 +1,306 @@
+local cluster = require('luatest.replica_set')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group('Persistent triggers on a single node')
+
+g.before_each(function()
+    g.server = server:new{
+        alias = 'default',
+    }
+    g.server:start()
+end)
+
+g.after_each(function()
+    g.server:drop()
+    g.server = nil
+end)
+
+g.test_recovery = function()
+    g.server:exec(function()
+        local trigger = require('trigger')
+        local max_i = 5
+        local event_name = 'persistent_event'
+        trigger.set(event_name, 'not_truly_persistent' .. tostring(max_i + 1),
+                    function() end)
+        for i = max_i, 1, -1 do
+            box.schema.func.create('truly_persistent' .. tostring(i), {
+                language = 'lua', body = 'function() end', trigger = event_name
```

Comment:
So it's not a trigger name - it's an event name, actually. Maybe, better call it `on_event`, not `trigger`?

Reply:
We also operate with term `event` in `box.watch`, and there we have keys, which are strings as well, so `on_event` will be confisung.
The idea of this name is that it refers to the module `trigger`.